docs/ai/how-to/app-service-entra-auth.md

```diff
@@ -0,0 +1,155 @@
+
+The [Semantic Kernel SDK](/semantic-kernel/overview) enables you to easily implement Microsoft Entra authentication in your .NET application.
+
+## Prerequisites
+
+* An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+* [.NET SDK](https://dotnet.microsoft.com/download/visual-studio-sdks)
+* [`Microsoft.SemanticKernel` NuGet package](https://www.nuget.org/packages/Microsoft.SemanticKernel)
+* [`Azure.Identity` NuGet package](https://www.nuget.org/packages/Azure.Identity)
+* [Create and deploy an Azure OpenAI Service resource](/azure/ai-services/openai/how-to/create-resource)
+* [Create and deploy a .NET application to App Service](/azure/app-service/quickstart-dotnetcore)
+
+## Add a managed identity to App Service
+
+### Add a system-assigned identity
+
+:::zone target="docs" pivot="azure-portal"
+
+1. Navigate to your app's page on the [Azure Portal](https://aka.ms/azureportal), scroll down to the **Settings** group.
+1. Select **Identity**.
+1. Within the **System assigned** tab, switch **Status** to On. Click **Save**.
+
+:::zone-end
+
+:::zone target="docs" pivot="azure-cli"
+
+Run the az webapp identity assign command to create a system-assigned identity:
+
+```azurecli
+az webapp identity assign --name <app-name> --resource-group <group-name>
+```
+
+:::zone-end
+
+### Add a user-assigned identity
+
+Creating an app with a user-assigned identity requires that you create the identity and then add its resource identifier to your app config.
+
+:::zone target="docs" pivot="azure-portal"
+
+1. Create a user-assigned managed identity resource according to these [instructions](/azure/active-directory/managed-identities-azure-resources/how-to-manage-ua-identity-portal#create-a-user-assigned-managed-identity).
+1. In the left navigation for your app's page, scroll down to the **Settings** group.
+1. Select **Identity**.
+1. Select **User assigned > Add**.
+1. Search for the identity you created earlier, select it, and select **Add**.
+
+    > [!NOTE]
+    > Once you select **Add**, the app restarts.
+
+:::zone-end
+
+:::zone target="docs" pivot="azure-cli"
+
+1. Create a user-assigned identity:
+
+    ```azurecli
+    az identity create --resource-group <group-name> --name <identity-name>
+    ```
+
+1. Assign the identity to your app:
+
+    ```azurecli
+    az webapp identity assign --resource-group <group-name> --name <app-name> --identities <identity-id>
+    ```
+
+:::zone-end
+
+## Add an Azure OpenAI user role to your managed identity
+
+:::zone target="docs" pivot="azure-portal"
+
+1. In the [Azure Portal](https://aka.ms/azureportal), navigate to the scope you want to grant **Azure OpenAI** access to.
+    1. This can be a **Management group**, **Subscription**, **Resource group**, or a specific **Azure OpenAI** resource.
```

Comment:
I just had it as an ordered list because the parent item was in an ordered list. I updated this sub-bullet to be a unordered list instead.

@gewarren in my local preview the formatting is a little off when I mix ordered and unordered list. Does it look alright in the Learn review link?

Reply:
It looks like this:

<img width="453" alt="image" src="https://github.com/dotnet/docs/assets/24882762/78385d5d-5c9b-45f8-b619-820bfc4ec720">

Reply:
That looks much better than my preview, thanks @gewarren!